### ISSUE TITLE
HA 2024.1.2 ELECTRIC_CURRENT_AMPERE ➜ UnitOfElectricCurrent.AMPERE

### ISSUE THREAD

**Issue by RedNo7** [gndean/home-assistant-hypervolt-charger#39]
Got the following in my logs after updating to 2024.1.0 (from 2023.12.4).

Log details:
```yaml
Logger: homeassistant.const
Source: helpers/deprecation.py:205
First occurred: 16:23:54 (2 occurrences)
Last logged: 16:23:54

ELECTRIC_CURRENT_AMPERE was used from hypervolt_charger, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfElectricCurrent.AMPERE instead, please create a bug report at https://github.com/gndean/home-assistant-hypervolt-charger/issues
ELECTRIC_POTENTIAL_VOLT was used from hypervolt_charger, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfElectricPotential.VOLT instead, please create a bug report at https://github.com/gndean/home-assistant-hypervolt-charger/issues
```

**Comment by gndean** [comment]
Thanks for the feedback. This should now be fixed in V1.3.1

**Comment by RedNo7** [comment]
Working fir me!
